editor/src/document/document_message_handler.rs

```diff
@@ -693,6 +693,7 @@ impl DocumentMessageHandler {
 					]],
 					selected_index: Some(self.document_mode as u32),
 					draw_icon: true,
+					interactive: false, // TODO: set to true when dialogs are not spawned
```

Comment:
I think it's preferable to have the comin soon dialog pop up if the user uses arrow keys (which is already unlikely) for consistency with other dropdown menus. I think the interactive should be always true for dropdown menus with the exception of rare cases where performance reasons require we disable it. So can you set this one to `true ` also, but instead of setting it here (and later in this file, and in export_dialog.rs) can we use `..Default::default()` and make that default to `true` for DropdownInputs?